.github/workflows/publish-dev-assets.yml

```diff
@@ -9,17 +9,41 @@ permissions: read-all
 
 jobs:
   build:
+    env:
+        AZURE_SUBSCRIPTION_ID: daae1e1a-63dc-454f-825d-b39289070f79
```

Comment:
wondering if any of these info ( sub/clientId/tenantId) should be secrets

Reply:
That's a good point. None of this is technically sensitive right? However even for ease of configuration without having to send PRs, I think it makes sense. There's other workflows I think that also have this issue.

Reply:
switched all workflows to use secrets and variables instead